index.html

```diff
@@ -0,0 +1,311 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Coding Theatre</title>
+    <script
+      src="https://kit.fontawesome.com/0ec53d8e70.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <!--Navigation-->
+    <nav class="container">
+      <div class="nav-left container">
+        <!--icons left-->
+        <i class="fa-solid fa-bars-staggered"></i>
+        <!--styled together-->
+        <img
+          src="./images/play.png"
+          alt="play"
+          class="play"
+          width="80"
+          height="80"
+        />
+        <span class="logo-name">MyTube</span>
+      </div>
+      <!--middle icons and search bar-->
+      <div class="nav-middle container">
+        <div class="search-box container">
+          <input type="text" placeholder="Search" />
+          <i class="fa-solid fa-magnifying-glass"></i>
+        </div>
+        <i class="fa-solid fa-microphone"></i>
+      </div>
+      <!--icons right-->
+      <div class="nav-right container">
+        <i class="fa-solid fa-square-plus"></i>
+        <i class="fa-solid fa-bell"></i>
+        <img src="./images/me.jpg" alt="me" class="user-icon" />
+      </div>
+    </nav>
```

Comment:
For higher SEO and semantic reasons, I think it would be better to wrap your top navigation inside the header tag.

Reply:
- [ ] Kindly make sure that when you are doing review kindly give more description on why you asked the student to follow **SEO semantic** rules and apply them in their projects. try to be more specific as much as possible. this way you let the student to better understand why there are changes requested. [more info](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md) :+1: :+1: